docs/sources/alerting/unified-alerting/high-availability.md

```diff
@@ -0,0 +1,62 @@
++++
+title = " Configure high availability"
+description = "High Availability"
+keywords = ["grafana", "alerting", "tutorials", "ha", "high availability"]
+weight = 450
++++
+
+# High availability
+
+The Grafana alerting systems has two main components, the `Scheduler` and the internal `Alertmanager`. The `Scheduler` is responsible for the evaluation of your [alert rules]({{< relref "./fundamentals/evaluate-grafana-alerts.md" >}}) while the internal Alertmanager takes care of the **routing** and **grouping**.
+
+When it comes to a highly available setup, the operational mode of the scheduler is unaffected. All alerts will be evaluated in every instance. The operational change happens in the Alertmanager, it is in charge of also **deduplicating** the alert notifications between Grafana instances.
+
+```
+  .─────.
+ ╱       ╲                                                                      ┌────────────────┐
+(  User   )──────┐                        ┌──────────────────────────────────┐  │                │
+ `.     ,'       │                        │┌─────────┐      ┌──────────────┐ │  │                ▼
+   `───'         │                        ││Scheduler│──────▶Alertmananager│─┼──┘    ┌──────────────────────┐
+                 │      ┌───────────┐  ┌─▶│└─────────┘      ▲──────────────┤ │       │                      │
+  .─────.        │      │   Load    │  │  │Grafana          │              │ │       │                      │
+ ╱       ╲       │      │ Balancing │  │  └─────────────────┼──────────────┼─┘       │     Integrations     │
+(  User   )──────┼─────▶│  Reverse  │──┤  ┌─────────────────┼──────────────┼─┐       │                      │
+ `.     ,'       │      │   Proxy   │  │  │┌─────────┐      ├──────────────▼ │       │                      │
+   `───'         │      └───────────┘  │  ││Scheduler│──────▶Alertmananager│─┼──┐    └──────────────────────┘
+                 │                     └─▶│└─────────┘      └──────────────┘ │  │                ▲
+  .─────.        │                        │Grafana                           │  │                │
+ ╱       ╲       │                        └──────────────────────────────────┘  └────────────────┘
+(  User   )──────┘
+ `.     ,'
+   `───'
+```
+
+The coordination between Grafana instances happens via [Gossip](https://en.wikipedia.org/wiki/Gossip_protocol). Alerts are not gossiped between instances. It is expected that each scheduler delivers the same alerts to each Alertmanager.
+
+The two types of messages that are gossiped between instances are:
+
+- Notification logs: Who (which instance) notified what (which alert)
+- Silences: If an alert should fire or not
+
+These two states are persisted in the database periodically and when Grafana is gracefully shutdown.
```

Comment:
```suggestion
Both notifications and silences are persisted to the database periodically and during graceful shutdown of Grafana.
```

Reply:
I think I prefer my phrasing on this one. I've just enumerated the states and this is within the same section. The persistence happens within the database which is what I have referred to.